tests/unit/iam_auth_tests.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+# Copyright (c) 2017 IBM. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Unit tests for IAM authentication. """
+import json
+import mock
+import unittest
+
+from cloudant._common_util import IAMSession
+
+
+class IAMAuthTests(unittest.TestCase):
+    """ Unit tests for IAM authentication. """
+
+    @mock.patch('cloudant._common_util.ClientSession.request')
+    def test_iam_get_access_token(self, m_req):
+        m_response = mock.MagicMock()
+        m_response.json.return_value = {'access_token': 'bar'}
```

Comment:
This mock response should probably have more of the expected token content in it.